Modules/Registration/Common/wrapping/itkPointSetToPointSetMetric.wrap

```diff
@@ -0,0 +1,10 @@
+itk_wrap_include("itkPointSet.h")
+UNIQUE(types "${WRAP_ITK_REAL};D")
+
+itk_wrap_class("itk::PointSetToPointSetMetric" POINTER_WITH_2_SUPERCLASSES)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
```

Comment:
Shouldn't this be `ITK_WRAP_DIMS` too?

Reply:
In [itkPointSetToPointSetMetricv4](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Registration/Metricsv4/wrapping/itkPointSetToPointSetMetricv4.wrap) we have used ITK_WRAP_IMAGE_DIMS. So I used the same here.